lottie/src/main/java/com/airbnb/lottie/parser/LottieCompositionMoshiParser.java

```diff
@@ -107,7 +107,7 @@ public static LottieComposition parse(JsonReader reader) throws IOException {
     Rect bounds = new Rect(0, 0, scaledWidth, scaledHeight);
 
     composition.init(bounds, startFrame, endFrame, frameRate, layers, layerMap, precomps,
-        images, Utils.dpScale(), characters, fonts, markers);
```

Comment:
Note that I do pass the unscaled width & height here.

Reply:
Can you make the names very clearly named `unscaledWidth` and `unscaledHeight`?